NetSDK/App.config

```diff
@@ -36,6 +36,9 @@
     <add key="TEST_IMPRESSIONS_URL_TEMPLATE" value="testImpressions/bulk" />
     <add key="METRICS_URL_TEMPLATE " value="metrics/{endpoint}" />
     <add key="SPLITS_STORAGE_CONCURRENCY_LEVEL" value="4" />
+    <add key="TEST_IMPRESSIONS_REFRESH_RATE" value=""/>
+    <add key="TEST_IMPRESSIONS_CACHE_SIZE_PER_TEST" value=""/>
```

Comment:
we may not need this. 

Mind explaining what this is used for? We are moving towards a simple list of impressions. In java we use an AtomicBlockingQueue